File: src/WebTodoList.ViewModels/Todo/ListItemViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebTodoList.ViewModels.Todo
{
    public class ListItemViewModel
    {
        public Guid Id { get; set; }

        public string Text { get; set; }

        public bool IsDone { get; set; }
    }
}

File: src/Clients/WebTodoList.Client.ReactJs/ClientApp/src/viewModels/ListItemViewModel.js
class ListItemViewModel {
    id;
    text;
    isDone;

    static map(id, text, isDone) {
        let viewModel = new ListItemViewModel();
        viewModel.id = id;
        viewModel.text = text;
        viewModel.isDone = isDone;

        return viewModel;
    }
}

export default ListItemViewModel;